tests/modules/configs/test_config.py

```diff
@@ -121,3 +120,134 @@ def test_get_conf_path_file_not_found(self):
             conf.get_conf_path(conf_path)
         assert ex.type == FileNotFoundError
         assert err_msg.get_conf_path.get("file_not_found") in str(ex.value)
+
+
+class TestMergedConf:
+    """Class to test functions of MergedConf Class"""
+
+    @staticmethod
+    def none_to_dict(obj):
+        if obj is None:
+            return {}
+        return obj
+
+    @pytest.mark.parametrize("dct, expect", [
+        [merge_conf.dct1, merge_conf.expect1],
+        [merge_conf.dct2, merge_conf.expect2],
+        [merge_conf.dct3, merge_conf.expect3],
+    ])
+    def test_merge_conf(self, dct, expect):
+        conf = MergedConf(Loggers({"debug": True}))
```

Comment:
Why do we need debug here?

Reply:
looks Loggers() only need a dict here, let me test again

Reply:
Ahhh, you are right! `Loggers.add_console_handler()` will throw KeyError if `debug` is undefined. I will fix that right away, so that empty dict is acceptable.

Reply:
debugs should be removed now, the issue is solved.